pkg/env/sensor.go

```diff
@@ -42,4 +42,7 @@ var (
 	// This is meant to be a 'kill switch' that allows for local scanning to continue (ie: for OCP internal repos)
 	// in the event the delegated scanning capabilities are causing unforeseen issues.
 	DelegatedScanningDisabled = RegisterBooleanSetting("ROX_DELEGATED_SCANNING_DISABLED", false)
+
+	// PingInterval defines in which interval Sensor should send ping messages to Central.
```

Comment:
nit: `PingInterval defines the time between each Sensor ping message to Central (in seconds).`

Reply:
Is this too frequent? From a very service-level search, I found both TCP and gRPC keep-alive intervals to be on the order of hours (2)

Reply:
So for the particular customer case that triggered this fix the timeout set was 60 seconds, so the ping interval should be less than that.

For example, [the keepalive for gRPC connections](https://github.com/stackrox/stackrox/blob/master/pkg/clientconn/client.go#L418-L422) is configured to be after 10s of not seeing communication, so 30 seconds seems fair here.

Reply:
Seems like the minimum is 10s, so anything greater than that seems to be fair game: https://pkg.go.dev/google.golang.org/grpc/keepalive#ClientParameters